src/orga/governance.rst

```diff
@@ -139,6 +139,7 @@ To call for a standard vote, here is a template PR comment:
          time.  In order for a timeout to occur, the vote must have:
 
              * been open for at least 2 weeks
+             * been presented and discussed at a Core team meeting
```

Comment:
We had that long before we created the Governance docs. It would be nice to have it again.
Not only we can call on people to vote but we can also answer clarification questions face-to-face. :+1: